ergo-wallet/src/main/scala/org/ergoplatform/wallet/interpreter/ErgoInterpreter.scala

```diff
@@ -27,7 +27,7 @@ class ErgoInterpreter(params: ErgoLikeParameters)(implicit IR: IRContext)
     * Check that expired box is spent in a proper way
     *
     * @param box           - box being spent
-    * @param output        - newly created box
+    * @param output        - newly created box when storage fee covered, otherwise any output box
```

Comment:
This is not related to the changes, but I cannot understand the logic of this method. 
Looks like something wrong, or comments would be helpfull.